### PR TITLE
Make Doctrine DBAL Connection configurable

### PR DESCRIPTION
**Pull request by wjzijderveld**
Follow up of #137

### CODE REVIEW

**Comment by wjzijderveld** [comment]
Also see #3

**Comment by othillo** [comment]
:+1: 
squash please

File: src/Broadway/Bundle/BroadwayBundle/Command/SchemaEventStoreCreateCommand.php [hunk 1]
```diff
@@ -11,7 +11,10 @@
 
 namespace Broadway\Bundle\BroadwayBundle\Command;
 
+use Broadway\EventStore\DBALEventStore;
 use Exception;
+use RuntimeException;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
```
Comment (fritsjanb) [inline src/Broadway/Bundle/BroadwayBundle/Command/SchemaEventStoreCreateCommand.php]: All 3 new `use`'s are unused :)

File: src/Broadway/Bundle/BroadwayBundle/Command/SchemaEventStoreDropCommand.php [hunk 2]
```diff
@@ -11,7 +11,10 @@
 
 namespace Broadway\Bundle\BroadwayBundle\Command;
 
+use Broadway\EventStore\DBALEventStore;
 use Exception;
+use RuntimeException;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
```
Comment (fritsjanb) [inline src/Broadway/Bundle/BroadwayBundle/Command/SchemaEventStoreDropCommand.php]: All 3 new `use`'s are unused :)

File: src/Broadway/Bundle/BroadwayBundle/DependencyInjection/BroadwayExtension.php [hunk 3]
```diff
@@ -11,6 +11,7 @@
 
 namespace Broadway\Bundle\BroadwayBundle\DependencyInjection;
 
+use InvalidArgumentException;
```
Comment (fritsjanb) [inline src/Broadway/Bundle/BroadwayBundle/DependencyInjection/BroadwayExtension.php]: unused

**Pull request merged**